### PR TITLE
channelz: fix GetSecurityValue function name.

### PR DESCRIPTION
**Pull request by lyuxuan**

### CODE REVIEW

File: test/channelz_test.go [hunk 1]
```diff
@@ -1196,6 +1200,53 @@ func TestCZServerSocketMetricsKeepAlive(t *testing.T) {
...
+		}
+		if len(sc.Sockets) != 1 {
+			return false, fmt.Errorf("there should only be one socket under subchannel %d, not %d", sc.ID, len(sc.Sockets))
+		}
+		for id = range sc.Sockets {
+			break
+		}
+		skt := channelz.GetSocket(id)
+		cert, _ := tls.LoadX509KeyPair(testdata.Path("server1.pem"), testdata.Path("server1.key"))
+		securityVal := skt.SocketData.Security.(*credentials.TLSChannelzSecurityValue)
```
Comment (menghanl) [inline test/channelz_test.go]: Do an `ok` check to avoid panic is `Security` is not the expected type.

File: test/channelz_test.go [hunk 2]
```diff
@@ -1196,6 +1200,53 @@ func TestCZServerSocketMetricsKeepAlive(t *testing.T) {
...
+		if !reflect.DeepEqual(securityVal.RemoteCertificate, cert.Certificate[0]) {
+			return false, fmt.Errorf("SocketData.Security.RemoteCertificate got: %v, want: %v", securityVal.RemoteCertificate, cert.Certificate[0])
+		}
+		for _, v := range credentials.CipherSuiteLookup {
```
Comment (menghanl) [inline test/channelz_test.go]: Are the expected suites a subset of the whole `CipherSuiteLookup`? I would prefer to not export the map for testing.

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* One or more commits are not associated with a GitHub user. The email used in the Git commit must match a verified email address in a GitHub user profile. For more information, please see https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/

_Regards,
CLA GitHub bot_

File: test/channelz_test.go (hunk 1 above)
Comment (lyuxuan) [reply test/channelz_test.go]: done

File: test/channelz_test.go (hunk 2 above)
Comment (lyuxuan) [reply test/channelz_test.go]: make a test copy of all the cipher suites for checking against.

File: test/channelz_test.go [hunk 3]
```diff
@@ -1196,6 +1200,82 @@ func TestCZServerSocketMetricsKeepAlive(t *testing.T) {
...
+
+func TestCZSocketGetSecurityValueTLS(t *testing.T) {
+	defer leakcheck.Check(t)
+	channelz.NewChannelzStorage()
+	e := tcpTLSRREnv
+	te := newTest(t, e)
+	te.startServer(&testServer{security: e.security})
+	defer te.tearDown()
+	te.clientConn()
+	if err := verifyResultWithDelay(func() (bool, error) {
```
Comment (menghanl) [inline test/channelz_test.go]: Not related to the changes in this PR.

Can this just be `verifyResultWithDelay(func() error)`? (No `bool` in return value).
It seems the function either return `true, nil` or `false, <non-nil>`.

Comment (lyuxuan) [reply test/channelz_test.go]: right. I think the original idea is that we may want `false, nil` sometimes to skip creating intermediate errors. But it seems we don't really have such use case yet. We probably should just refactor to what you suggested.

**Pull request merged**